rules/windows/discovery_high_number_ad_properties.toml

```diff
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2023/01/29"
+integration = ["windows", "endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/01/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identify read access to a high number of Active Directory object attributes. The knowledge of objects properties 
+can help adversaries find vulnerabilities, elevate privileges or collect sensitive information.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-system.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Suspicious Access to LDAP Attributes"
+note = """## Setup
+## Setup
```

Comment:
```suggestion
## Setup

```

Reply:
`setup` is now it's own independent rule field so we can remove it from note.